lib/faker/datetime.ex

```diff
@@ -0,0 +1,45 @@
+if Version.match?(System.version(), ">= 1.3.0") do
+  defmodule Faker.DateTime do
+    @microseconds_per_day 86400000000
+
+    @doc """
+    Returns a random date in the past up to N days, today not included
+    """
+    @spec backward(integer) :: DateTime.t
+    def backward(days) do
+      forward(-days)
+    end
+
+    @doc """
+    Returns a random date in the future up to N days, today not included
+    """
+    @spec forward(integer) :: DateTime.t
+    def forward(days) do
+      unix_from = DateTime.utc_now() |> DateTime.to_unix(:microseconds)
+
+      unix_between(unix_from, days)
+    end
+
+    @doc """
+    Returns a random date between two dates, start date and end date not
```

Comment:
Do you think the wording is confusing? I figured the typespec would make it clear what type is being returned.